tests/scalar.rs

```diff
@@ -0,0 +1,18 @@
+use momento::requests::cache::basic::set::Set;
+use momento_test_util::CACHE_TEST_STATE;
+use uuid::Uuid;
+
+#[tokio::test]
+async fn set_happy_path() {
+    let client = CACHE_TEST_STATE.client.clone();
+    let cache_name = CACHE_TEST_STATE.cache_name.clone();
+    let key = &Uuid::new_v4().to_string();
+    let value = &Uuid::new_v4().to_string();
+
+    let result = client
+        .set(cache_name.clone(), key.clone(), value.clone())
```

Comment:
Is it necessary to clone the cache_name, key, and values here?

Reply:
it shouldn't be; if it is then our API needs improvement :P

Reply:
`cache_name` wasn't needed; but see an error if i don't clone other two:

```
 ---                  ^^^^^ the trait `From<&String>` is not implemented for `Vec<u8>`, which is required by `&String: IntoBytes`
```

Reply:
The examples and tests for the existing APIs all clone the arguments, so I followed that pattern. I don't have a good instinct for what would be idiomatic here. Do rust functions of this sort generally expect to take ownership of the arguments they are given?

Reply:
they do take onwership, but if you pass a `&str` then the ownership is just of a reference so you can still use it. i merged a PR this morning that cleaned up tests a bit to store some vars of type `&str` to make the tests a little simpler (i hope).